bin/cache_clearing_service

```diff
@@ -1,5 +1,2 @@
-#!/usr/bin/env ruby
-
-require_relative "../app/environment"
-
-Listener.call
+#!/usr/bin/env bash
+rake message_queue:consumer
```

Comment:
Do we still need this file?  Not sure what the benefit is over calling rake directly.

Reply:
I left this so that deploying the app wouldn't break for now. I agree it's pretty redundant and will raise a follow up PR once this and puppet are deployed and working to remove it.

Reply:
👍